### ISSUE TITLE
Labels should be beside controls

### ISSUE THREAD

**Issue by kousu** [santropolroulant/volunteerdb#21]
In the CakePHP 3 version (#1) labels on /volunteers/edit are now above all their controls, instead of beside them. Presumably I messed up the CSS in using FormHelper.

**Comment by kousu** [comment]
I think I need to add this CSS:

```
.control-group label {

    float: left;
    width: 160px;
    padding-top: 5px;
    text-align: right;
}

.control-group input {

    margin-left: 10px;

}
```

**Comment by kousu** [comment]
Fixed in c7ac87754ca54a249aec38aada4a9908c09d332a
